lib/apivore/rspec_matchers.rb

```diff
@@ -35,13 +35,25 @@ module RspecMatchers
       end
     end
 
-    matcher :be_consistent_with_swagger_definitions do |master_swagger|
+    matcher :be_consistent_with_swagger_definitions do |master_swagger, current_service|
 
       attr_reader :actual, :expected
 
+      def cleaned_definitions(definitions, current_service)
+        definitions.transform_values do |definition_fields|
+          # We ignore definitions that are owned exclusively by the current_service
+          if [current_service] == definition_fields['x-services']
+            nil
+          else
+            # 'x-services' is added by api.westfield.io - services shouldn't need to define it
```

Comment:
Is this comment needed here. `cleaned_definitions` is being applied to the master_swagger doc so it is expected to have the x-services extension